libs/sdk-core/src/greenlight/node_api.rs

```diff
@@ -1746,10 +1751,30 @@ impl NodeAPI for Greenlight {
         let mut result = Map::new();
         for command in all_commands {
             let command_name = command.clone();
-            let res = self
+            let mut res = self
                 .execute_command(command)
                 .await
                 .unwrap_or_else(|e| json!({ "error": e.to_string() }));
+            
+            // Convert any byte arrays to hex in the response
+            if let Value::Object(obj) = &mut res {
+                for (_, value) in obj.iter_mut() {
+                    if let Value::Array(arr) = value {
+                        for item in arr.iter_mut() {
+                            if let Value::Object(item_obj) = item {
+                                for (_, v) in item_obj.iter_mut() {
+                                    if let Value::String(s) = v {
+                                        if let Ok(bytes) = hex::decode(s) {
+                                            *v = Value::String(hex::encode(bytes));
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+            
```

Comment:
This is already fixed in #1091? 
The objects from execute_dev_command are printed in hex already. What am I missing?